src/components/AlexCard.jsx

```diff
@@ -1,5 +1,5 @@
 import React from "react"
-import {Row, Col} from "reactstrap"
+// import {Row, Col} from "reactstrap"
```

Comment:
refactor when all the components are done. Delete the unused/commented out code.